templates/includes/_info_messages.html

```diff
@@ -0,0 +1,8 @@
+ {% for message in messages %}
+    <div class="alert alert-info alert-dismisable" role="alert" style="margin-top:10px;margin-bottom:10px;">
```

Comment:
ups, it changed something, but then I moved it into another part of template, so now it is useless, I will get rid of it.